core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -718,23 +718,15 @@ public static function isVisibleElement(Model $objElement)
 		$blnReturn = true;
 
 		// Only apply the restrictions in the front end
-		if (TL_MODE == 'FE')
+		if (TL_MODE == 'FE' && $objElement->protected)
 		{
+			$blnReturn = false;
 			$blnFeUserLoggedIn = System::getContainer()->get('contao.security.token_checker')->hasFrontendUser();
+			$groups = StringUtil::deserialize($objElement->groups);
```

Comment:
```suggestion
			$groups = StringUtil::deserialize($objElement->groups, true);
```
Otherwise the `in_array` generates a warning.

Reply:
Good catch. 👍🏻

Reply:
See #3136.